airflow/providers/google/cloud/transfers/bigquery_to_mssql.py

```diff
@@ -111,47 +109,23 @@ def __init__(
         self.batch_size = batch_size
         self.location = location
         self.impersonation_chain = impersonation_chain
+        try:
+            _, self.dataset_id, self.table_id = source_project_dataset_table.split('.')
+        except ValueError:
+            raise ValueError(f'Could not parse {source_project_dataset_table} as <project>.<dataset>.<table>')
```

Comment:
```suggestion
            raise ValueError(f'Could not parse {source_project_dataset_table} as <project>.<dataset>.<table>') from None
```

to hide the original exception raised by `split()`. This makes the traceback cleaner.

Reply:
Updated